services/manifest-repo-export-service/pkg/repository/transformer_test.go

```diff
@@ -1990,3 +1994,735 @@ func TestLocks(t *testing.T) {
 		})
 	}
 }
+
+////@@@@@@@@@@
+
+func TestCreateUndeployLogic(t *testing.T) {
+	const appName = "app1"
+	const authorName = "testAuthorName"
+	const authorEmail = "testAuthorEmail@example.com"
+	tcs := []struct {
+		Name            string
+		Transformers    []Transformer
+		expectedData    []*FilenameAndData
+		expectedMissing []*FilenameAndData
+		expectedMessage string
+		expectedError   error
```

Comment:
These two doesn't show up in any test cases in this test, So maybe we can remove them